backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/deliveryplanning/DeliveryPlanningCreateRequest.java

```diff
@@ -119,6 +119,12 @@
 	@NonNull
 	Quantity actualDeliveredQty;
 
+	@NonNull Quantity plannedLoadedQty;
+
+	@NonNull Quantity plannedDischargeQty;
```

Comment:
i think i get  `plannedDischargeQty` and `actualDischargeQty`, but then what is `actualDeliveredQty`?
i mean what's the difference between `actualDeliveredQty` and `actualDischargeQty` ?

Reply:
> what's the difference between `actualDeliveredQty` and `actualDischargeQty` ?

Asked @metasnw

Reply:
@metas-ts depending on the incoterm the good issues (in our language "shipment") is either when the goods are loaded to  a ship or discharged at a warehouse.
Nevertheless we want to save both qtys in case they change (they do actually) from loading to discharge.

Reply:
thanks